cmd/mdatagen/internal/samplescraper/internal/metadata/generated_config_test.go

```diff
@@ -27,11 +27,42 @@ func TestMetricsBuilderConfig(t *testing.T) {
 			name: "all_set",
 			want: MetricsBuilderConfig{
 				Metrics: MetricsConfig{
-					DefaultMetric:            MetricConfig{Enabled: true},
-					DefaultMetricToBeRemoved: MetricConfig{Enabled: true},
-					MetricInputType:          MetricConfig{Enabled: true},
-					OptionalMetric:           MetricConfig{Enabled: true},
-					OptionalMetricEmptyUnit:  MetricConfig{Enabled: true},
+					DefaultMetric: MetricConfig{
+						Enabled:             true,
+						AggregationStrategy: "sum"},
```

Comment:
We should generate `AggregationStrategySum` instead of `"sum"`
```suggestion
					DefaultMetric: MetricConfig{
						Enabled:             true,
						AggregationStrategy: AggregationStrategySum,
					},
```